services/web-client/source/class/qxapp/components/workbench/SettingsView.js

```diff
@@ -47,35 +46,34 @@ qx.Class.define("qxapp.components.workbench.SettingsView", {
       titleBox.add(doneBtn);
       this.add(titleBox);
 
-      let scope = this;
       doneBtn.addListener("execute", function() {
-        scope.fireEvent("SettingsEditionDone");
-      }, scope);
+        this.fireEvent("SettingsEditionDone");
+      }, this);
     },
 
-    _InitSettings: function() {
-      this._settingsBox = new qx.ui.container.Composite(new qx.ui.layout.VBox(10));
-      this.add(this._settingsBox);
+    __InitSettings: function() {
```

Comment:
naming consistency? should not be ``__initSettings``

Reply:
Danke